contracts/eos/BancorConverter/BancorConverter.hpp

```diff
@@ -126,6 +126,35 @@ CONTRACT BancorConverter : public eosio::contract { /*! \endcond */
 
             }; /** @}*/
 
+        /**
+         * @defgroup BancorConverter_Reserves_Table Reserves Table
+         * @brief This table stores the reserve balances and related information for the reserves of every converter in this contract
+         * @details SCOPE of this table is the converters' smart token symbol's `code().raw()` values
+         * @{
+         *//*! \cond DOCS_EXCLUDE */
+            TABLE reserve_v2_t {
+                /**
+                 * @brief symbol of the smart token -- representing a share in the reserves of this converter
+                 * @details PRIMARY KEY for this table is `currency.code().raw()`
+                 */
+                symbol currency;
+
+                /**
+                 * @brief reserve ratios relative to the other reserves
+                 */
+                map<symbol_code, uint64_t> ratios;
```

Comment:
the correct term today is `weights`.
btw if it's possible and has no negative implications, it's better to use 'reserve_ratios','reserve_balances'.

Reply:
👍 for sure, we can do that

There's no side effects for changing naming conventions

The only time it breaks things is whenever you change the data type.

You can change row names at anytime.

Reply:
Using `map<symbol_code, uint64_t> reserve_weights` in new converter schema